pages/nft-book-store/collection/send/[collectionId].vue

```diff
@@ -160,6 +167,9 @@ import { useNftStore } from '~/stores/nft'
 import { useCollectionStore } from '~/stores/collection'
 import { parseImageURLFromMetadata } from '~/utils'
 import { getNFTs, getNFTOwner, signExecNFTSendAuthz, signSendNFTs } from '~/utils/cosmos'
+import { useMessageCharCount } from '~/composables/useMessageCharCount'
+
+const AUTHOR_MESSAGE_LIMIT = 98
```

Comment:
Should this be in `constant`?

Reply:
[♻️ Declare AUTHOR_MESSAGE_LIMIT as a constant](https://github.com/likecoin/nft-book-press/pull/212/commits/bb6e73edf3347bad3097cf1e3eda128e2ebd25e3)